keras_cv/applications/object_detection/retina_net/__internal__/utils/anchor_box.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+import keras_cv
+
+
+class AnchorBox(tf.keras.layers.Layer):
+    """Generates anchor boxes.
+
+    This class has operations to generate anchor boxes for feature maps at
+    strides `[8, 16, 32, 64, 128]`.
+
+    Args:
+        bounding_box_format:   The format of bounding boxes of input dataset. Refer
+            https://github.com/keras-team/keras-cv/blob/master/keras_cv/bounding_box/converters.py
+            for more details on supported bounding box formats.
+        aspect_ratios: A list of float values representing the aspect ratios of
+            the anchor boxes at each location on the feature map
+        scales: A list of float values representing the scale of the anchor boxes
+            at each location on the feature map.
+        num_anchors: The number of anchor boxes at each location on feature map
+        areas: A list of float values representing the areas of the anchor
```

Comment:
the order of docstring for this and strides are different from the param list below.